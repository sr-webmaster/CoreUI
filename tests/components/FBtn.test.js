import { mount } from '@vue/test-utils'
import Component from '@freshinup/core-ui/src/components/FBtn.vue'

describe('FBtn', () => {
  it('snapshot default', async () => {
    const wrapper = mount(Component)
    expect(wrapper.element).toMatchSnapshot()
  })
  it('snapshot with color', async () => {
    const wrapper = mount(Component, {
      propsData: {
        color: 'primary'
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
