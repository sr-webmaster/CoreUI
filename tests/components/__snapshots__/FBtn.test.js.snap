// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FBtn snapshot default 1`] = `
<button
  class="v-btn theme--light undefined-text--text"
  type="button"
>
  <div
    class="v-btn__content"
  />
</button>
`;

exports[`FBtn snapshot with color 1`] = `
<button
  class="v-btn theme--light primary primary-text--text"
  type="button"
>
  <div
    class="v-btn__content"
  />
</button>
`;
